.github/workflows/sync_s3_cache.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow uploads third-party dependencies to OSSCI S3.
+# For more information and documentation, see:
+# * https://github.com/pytorch/builder/main/thirdparty_deps/README.md
+# * https://github.com/pytorch/builder/pull/1096
+# * .github/pytorch-cached-s3-dependencies.yml
+
+name: Update S3 dependencies on public OSSCI S3 buckets
+on:
+  schedule:
+    # Sync every hour
+    - cron: "*/60 * * * *"
+  # Have the ability to trigger this job manually using the API as well
+  workflow_dispatch:
+
+jobs:
+  sync-s3-cache:
+    runs-on: ubuntu-18.04
+    if: ${{ github.repository == 'pytorch/pytorch' }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
```

Comment:
Minor: I think we are using https://github.com/pytorch/pytorch/tree/master/.github/actions/checkout-pytorch for quiet checkout. You can also set submodules to false and fetch-depth to 1 there.